### PR TITLE
feat: enable haproxy seamless reload

### PR DESCRIPTION
**Pull request by skygeo**

### CODE REVIEW

File: flyby/haproxy.py [hunk 1]
```diff
@@ -111,6 +111,11 @@ def _wait_pid(p):
             # Reload haproxy
             logger.info("Reloading HAProxy...")
             command = self.command + ["-sf", str(Haproxy.process.pid)]
+            # stave the state
```
Comment (gjtempleton) [inline flyby/haproxy.py]: `Save` not `stave`

File: flyby/config/haproxy.cfg.j2 [hunk 2]
```diff
@@ -71,6 +73,7 @@ backend flyby-admin
 
 {% for service  in services -%}
 backend {{ service.name }}-backend
+  load-server-state-from-file global
```
Comment (gjtempleton) [inline flyby/config/haproxy.cfg.j2]: Do we need this line if it's already set as a default?

Comment (skygeo) [inline flyby/config/haproxy.cfg.j2]: nop...removed

File: flyby/haproxy.py (hunk 1 above)
Comment (skygeo) [inline flyby/haproxy.py]: fixed

**Comment by gjtempleton** [comment]
LGTM

**Pull request merged**